# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/spd-5118_parity_4_4.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.3/include/asm-generic/bitops.h.vanilla	2023-06-05 18:02:33.758661000 +0200
+++ linux-6.3/include/asm-generic/bitops.h	2023-06-05 18:02:23.994659921 +0200
@@ -28,6 +28,7 @@
 #include <asm-generic/bitops/sched.h>
 #include <asm-generic/bitops/ffs.h>
 #include <asm-generic/bitops/hweight.h>
+#include <asm-generic/bitops/parity.h>
 #include <asm-generic/bitops/lock.h>
 
 #include <asm-generic/bitops/atomic.h>
--- linux-6.3/arch/sparc/include/asm/bitops_32.h.vanilla	2023-06-05 16:45:06.612147502 +0200
+++ linux-6.3/arch/sparc/include/asm/bitops_32.h	2023-06-05 16:45:49.172152205 +0200
@@ -99,6 +99,7 @@
 #include <asm-generic/bitops/__fls.h>
 #include <asm-generic/bitops/fls64.h>
 #include <asm-generic/bitops/hweight.h>
+#include <asm-generic/bitops/parity.h>
 #include <asm-generic/bitops/lock.h>
 #include <asm-generic/bitops/le.h>
 #include <asm-generic/bitops/ext2-atomic.h>
--- linux-6.3/arch/sparc/include/asm/bitops_64.h.vanilla	2023-06-05 16:45:52.003152518 +0200
+++ linux-6.3/arch/sparc/include/asm/bitops_64.h	2023-06-05 16:50:00.914180022 +0200
@@ -49,6 +49,7 @@
 unsigned int __arch_hweight8(unsigned int w);
 
 #include <asm-generic/bitops/const_hweight.h>
+#include <asm-generic/bitops/parity.h>
 #include <asm-generic/bitops/lock.h>
 #endif /* __KERNEL__ */
 
